src/components/commentInfo/CommentInfo.tsx

```diff
@@ -1,14 +1,67 @@
+import { useEffect, useState } from 'react';
+import { FIVE_SECONDS, LIKE, ONE_MINUTE, REPLY } from '../../constants';
+import { classes } from './CommentInfo.st.css';
+
 interface ICommentInfo {
-    creationDate: Date;
-    likesCount: number;
+    creationDate: number;
+    LikedBy: string[];
 }
 
-const CommentInfo = ({ creationDate, likesCount }: ICommentInfo) => {
+const CommentInfo = ({ creationDate, LikedBy }: ICommentInfo) => {
+    const SecondsPassedFromCreation = Math.floor((Date.now() - creationDate) / 1000);
+    const timePastFromCreation =
+        SecondsPassedFromCreation > 3599
+            ? `${Math.floor(SecondsPassedFromCreation / 3600)} h`
+            : SecondsPassedFromCreation > 59
+            ? `${Math.floor(SecondsPassedFromCreation / 60)} m`
+            : `${SecondsPassedFromCreation} s`;
+    const [timeAgo, setTimeAgo] = useState<string>(timePastFromCreation);
+
+    useEffect(() => {
+        const secondesInterval = setInterval(updateTimeAgo, FIVE_SECONDS);
+        let minutesInterval: number;
+        setTimeout(() => {
+            clearInterval(secondesInterval);
+            minutesInterval = setInterval(updateTimeAgo, ONE_MINUTE);
+        }, ONE_MINUTE);
+        return () => {
+            clearInterval(secondesInterval);
+            clearInterval(minutesInterval);
+        };
+    }, []);
+
+    const updateTimeAgo = () => {
+        setTimeAgo(getTimeAgo(creationDate));
+    };
+
+    const getTimeAgo = (creationDate: number) => {
+        const SecondsPassedFromCreation = Math.floor((Date.now() - creationDate) / 1000);
```

Comment:
We talked about changing this.

count the time passed in seconds instead of saving each type of time count.
export as much of this logic as you can to external helpers.

Reply:
instead of saving the values, I should calculate the values on the spot?